website/docs/d/organization.html.markdown

```diff
@@ -33,4 +33,5 @@ In addition to all arguments above, the following attributes are exported:
 * `cost_estimation_enabled` - Whether or not the cost estimation feature is enabled for all workspaces in the organization. Defaults to true. In a Terraform Cloud organization which does not have Teams & Governance features, this value is always false and cannot be changed. In Terraform Enterprise, Cost Estimation must also be enabled in Site Administration.
 * `owners_team_saml_role_id` - The name of the "owners" team.
 * `send_passing_statuses_for_untriggered_speculative_plans` - Whether or not to send VCS status updates for untriggered speculative plans. This can be useful if large numbers of untriggered workspaces are exhausting request limits for connected version control service providers like GitHub. Defaults to true. In Terraform Enterprise, this setting has no effect and cannot be changed but is also available in Site Administration.
-* `default_project_id` - ID of the organization's default project. All workspaces created without specifying a project ID are created in this project.
\ No newline at end of file
+* `aggregated_commit_status_enabled` - Whether or not to enable Aggregated Status Checks. This can be useful for monorepo repositories with multiple workspaces receiving status checks for events such as a pull request. If enabled, `send_passing_statuses_for_untriggered_speculative_plans` needs to be false.
```

Comment:
> If enabled, `send_passing_statuses_for_untriggered_speculative_plans` needs to be false.

nit: Is this relevant for the data source?

Reply:
Good catch. I guess since it's readonly. Will omit 👍